eng/versioning.targets

```diff
@@ -157,13 +157,13 @@
       WriteOnlyWhenDifferent="true" />
 
   </Target>
-
-  <!-- Removes assembly level attributes for Browser. -->
-  <Target Name="RemoveSupportedPlatformAssemblyAttributeForBrowser" AfterTargets="GetAssemblyAttributes" Condition="'$(TargetFrameworkSuffix)' == 'Browser' or '$(IsTestProject)' == 'true'">
+  
+  <!-- Removes assembly level attributes from test projects. -->
+  <Target Name="RemoveSupportedOSPlatformAttributeFromTestProjects" AfterTargets="GetAssemblyAttributes" Condition="'$(IsTestProject)' == 'true'">
```

Comment:
@buyaa-n Remind me... Did we already file an issue against the SDK that they should support a property that prevents the `[assembly:SupportedOSPlatform]` attribute from being emitted?  Once such a property exists, we'll want to update our build to set that property for test projects (rather than using a target to remove it after it's added).

Reply:
Yes we did, here it is https://github.com/dotnet/sdk/issues/14836